Packs/TrendMicroEmailSecurity/Integrations/TrendMicroEmailSecurityEventCollector/README.md

```diff
@@ -0,0 +1,47 @@
+Palo Alto Networks Trend Micro Email Security Event Collector integration for XSIAM.
+
+## Configure Trend Micro Email Security Event Collector on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Trend Micro Email Security Event Collector.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | Service URL |  | True |
+    | USER NAME |  | True |
+    | API Key |  | True |
+    | Trust any certificate (not secure) |  | False |
+    | Use system proxy settings |  | False |
+    | Hide sensitive details from email |  | False |
+    | The maximum number of events per fetch. | The maximum number of events to fetch every time fetch is executed. | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+**Note**: There are three types of fetches that the integration fetches, when the max fetch parameter is set to 1000 then 1000 logs will be retrieved from each type so that a total of 3000 logs can be retrieved.
```

Comment:
```suggestion
**Note**: There are three types of fetches that the integration fetches. When the max fetch parameter is set to 1000 then 1000 logs will be retrieved from each type so that a total of 3000 logs can be retrieved.
```
Regarding this wording:
"There are three types of fetches..."
I don't understand.  Can you list the different types of "fetches"? Should it be ..."three types of events"...?

Reply:
Yes, these are three types of events